### DISCUSSION TITLE
Asset file import

### DISCUSSION THREAD

**Discussion by Richard-L** [LostArtefacts/TRX#247] [General]
Some thoughts on importing the game's assets from genuine copies or dumps. Currently you're pointing to TombATI on the project's landing page which includes copyrighted audio assets someone ripped from an English PS1/Saturn release, as well as the few extra ambient tracks ripped from a PC release, all included as `.flac` files. From a strict point of view this is piracy. I would encourage against such affiliation, and I understand your goal is the removal of any TombATI depency or relation, anyway. As big a franchise it is, you can still legally buy the game even today on either [GOG](https://www.gog.com/game/tomb_raider_123) or [Steam](https://store.steampowered.com/app/224960/Tomb_Raider_I/). Supposedly you can also still buy the game on the PS3 store and emulate the PS1 version.

I understand you still use some code from TombATI currently, beyond just assets. But whenever you can, I would remove the TombATI link and instead advise to copy required assets from whatever sources people have, may that be a dump/image of their physical discs, the discs themselves, files obtained through Steam/GOG, files people may have from the android release if they bought it (the files are identical to PC retail), or even a TR1 PC demo of which several still circulate online. As you'll be aware the demo contains City of Vilcabamba's `LEVEL2.PHD` and the main menu. Else, for the lazy who need an OOBE experience, they could at least try TR1M at hand of Unfinished Business's files which are freeware.

Other than the legitimacy concern, the game has a great wealth of different asset versions, being one of the bigger productions of the 90s, and being released on three main platforms simultaneously, each receiving demos and pre-releases prior. It is one of the larger topics of fan interest to research these asset versions and compare their differences. In fact it opens up a bigger scope beyond just playing the game, one of enabling to use assets from different versions and centrally maintaining and organising them within TR1M. I'm aware this is likely too big an undertaking and not interesting to you, but I find at least worth mentioning. In fact, OpenLara has started doing this already, so I'd hope there might be room for it inside TR1M, too.

But to start, it would be enough to copy files originating from one of the retail PC releases. A very nice and quite extensive overview of them can be found here: https://secretsrunner.web.fc2.com/data_top/tr_top.html

I have two retail releases myself, both with these files in `\DATA`:

| Filename | MD5 |
| --- | --- |
| CRED1.PCX | 6272C2A557CD7597EFC6594E86AE7F2F |
| CRED2.PCX | 4497435DD41ABD9A247BE6AFBF264769 |
| CRED3.PCX | 9DF86E1D2523E3053D07B51CEBF33143 |
| CUT1.PHD | 1041EF2F23F62A740BFA27DF4AB638E5 |
| CUT2.PHD | F5D9CF5A7BB77676D59296D2038BB326 |
| CUT3.PHD | 636AF1916E950924AA3ACB51D2E47010 |
| CUT4.PHD | D2F20F38289A074A103AC9806B0DCD77 |
| EIDOSPC.PCX | B2815D0CC7E8DA83E0A2033A88943D82 |
| END.PCX | 8B678E0162D158C28B7911EF091C6C41 |
| GYM.PHD | 709C060B519862E1666B31562D505684 |
| INSTALL.PCX | 2F4F3EAFF7C99D9F16A530AFB4ECC3B1 |
| LEVEL1.PHD | E4A7F5CB5B70AB5F263FFF6E137FE0E3 |
| LEVEL2.PHD | C2481F4CD320EE5212793DC42DA22281 |
| LEVEL3A.PHD | 62FDE61721CA081BFFEF22360AA9273F |
| LEVEL3B.PHD | 9C071D8285C33C33C657B4EE7D8C9A20 |
| LEVEL4.PHD | 854584F54D98E99769F4D0313DB6B5BB |
| LEVEL5.PHD | AAB0D52E3FE6160DF6B05254449C6540 |
| LEVEL6.PHD | DF3512869F0E8DE3202C125AB2B5FC4E |
| LEVEL7A.PHD | 06D472646A70C8FDDC0F9B5561F111AB |
| LEVEL7B.PHD | 44DB81E97FA70DF98222A868CC460A62 |
| LEVEL8A.PHD | 5128696A7F7A742A2E10E5247FF7A84C |
| LEVEL8B.PHD | 30BA37005FE7C529C3A6CEE5618F5B08 |
| LEVEL8C.PHD | DF49B04B95C9923E5E7DA1095A07E486 |
| LEVEL10A.PHD | 4856B810C97398922283775EFE343AF5 |
| LEVEL10B.PHD | 25DCE79DFC83C5645E6413B80D93E25C |
| LEVEL10C.PHD | FBE5F4E588A90FE23865360C0A3B2C7E |
| TITLE.PHD | 12768DE79512EC9A2E4C49014378666B |
| TITLEH.PCX | 44C7DA52B6DD400F614C7845E6F05DA3 |

The CD also contains audio track assets of the opening theme played with the title screen, audio tracks of the voices and music during cutscenes, and the PC-only ambient tracks. How feasible would it be for TR1M to rip these during an initial setup dialogue?

Further, there are the PC versions of the FMVs as `.RPL`, which I understand aren't all supported yet.

The retail TRUB files I have are:

| Filename | MD5 |
| --- | --- |
| END2.tub | 84DF21D3C2ABD4E36B72EC9F030B1DE3 |
| install.pcx | B0C288E1B33284543C1D0036254DF90E |
| TITLE.PHD | 9BECC9EFBDDD9B24D4687643135EEA49 |
| TITLE.tub | 6272C2A557CD7597EFC6594E86AE7F2F |
| TITLEH.PCX | 4497435DD41ABD9A247BE6AFBF264769 |
| CAT.PHD | 9DF86E1D2523E3053D07B51CEBF33143 |
| CAT.tub | C1F883E4D0B49B34BBE2FF74E81E8ADB |
| cred0.pcx | 9CE02AD3D29F892816BF99047BCDACB4 |
| CRED1.PCX | B2815D0CC7E8DA83E0A2033A88943D82 |
| CRED2.PCX | 8B678E0162D158C28B7911EF091C6C41 |
| CRED3.PCX | 7F7C535BCAB001698A1F965053972F49 |
| EGYPT.PHD | D9812D95E8BDA3E3EA359B2D1D678D6C |
| EGYPT.tub | F03F7A545330D5BA30E2AEC62D47B4A3 |
| EIDOSPC.PCX | 243E3BCE0300E7FCFFB7BC2069DB843C |
| END.PCX | 2F4F3EAFF7C99D9F16A530AFB4ECC3B1 |
| END.PHD | 12768DE79512EC9A2E4C49014378666B |
| END.tub | 12768DE79512EC9A2E4C49014378666B |
| END2.PHD | 3DFBCE6C717400AC8AE9241D3FD25D9C |

Where it seems the `.tub` are unnecessary (?), in the case of the level "END" (Atlantean Stronghold) also apparently perfect duplicates.
 
I amassed 9 TR1 demos which all come with these four files, usually (but not always) matching with the later retail version like so:

| Filename | MD5 |
| --- | --- |
| INSTALL.PCX | 2F4F3EAFF7C99D9F16A530AFB4ECC3B1 |
| LEVEL2.PHD | C2481F4CD320EE5212793DC42DA22281 |
| TITLE.PHD | 12768DE79512EC9A2E4C49014378666B |
| TITLEH.PCX | 44C7DA52B6DD400F614C7845E6F05DA3 |

I would propose to test all the different versions we can find with TR1M in order to maximise compatibility with whatever people have. Also to counter things like #227.

Another step would be to add the fuller range of assets unique to the console releases, mainly the full soundtrack (both PS1 and Saturn), higher quality FMVs (PS1 release), loading screens, localisations of Lara's voice in the gym level (these are all audio tracks on PS1/Saturn), etc. I won't get into these for now, but just for an overview, these were the major releases sans later re-bundles as known to [redump.org](http://redump.org) for both platforms:

| Platform | Releases |
| --- | --- |
| PS1 | `SLES-00024` (UK), `SLES-00485` (FR), `SLES-00486` (DE), `SLUS-00152` (US), `SLPS 00617` (JP) |
| Saturn | `MK81086-50` (EU), `T-7910H` (US), `ST-6010G` (JP) |

And for further reference's sake, if you decided to add support to non-retail versions to allow access to some of their unique assets (different Lara models and textures, unused animations, different sound effects, etc), here's a nice selection of known [demos for all three platforms](https://www.tombraiderforums.com/showthread.php?t=207170) assembled by b122251.

A later feature request would be to offer an asset importer tool with decent UX that parses provided files and unlocks respective aspects of TR in TR1M.

I hope at least some of this is helpful.

**Comment by rr-** [comment]
Regarding the piracy part: you got this the other way around. We do not redirect people to TombATI because it offers some UB assets and we get some code as a bonus. It's the other way around: currently TR1M absolutely *requires* TombATI binaries (.exe, .dll) to even run. If we shipped some stripped down version of TombATI without the UB assets… it would still be illegal, because it still requires proprietary DLLs to play the FMVs and it is not possible to just not include the DLLs. The game will crash.

This issue consists of actually three parts…

- Kill TombATI dependency. This is what we've been doing with all the recent refactors, OpenGL and ffmpeg branches.
- Bundle legal assets from the freeware copy of UB.
- Create some kind of importer as a separate project unrelated to TR1M.

Before anything can happen, we need to finish the work on the first point which is essentially release 2.0 and is in progress. So far we've been getting buried under feature requests but fortunately this has calmed down recently, so we can continue focusing on that.

>Comparison tables

I find it funny that these detailed comparisons do not include the details from the most popular modern options to get the game: GOG and Steam.

>Lazy people

They're not lazy. It is generally quite sensible to expect things to just work. Not everyone has the will or time to fiddle with five registry patches and three tutorials to install a game.

>Piracy (again)

Steam version is bastardized and does not contain music assets, even when people paid money for the "full" version of the game, often for the nth time after buying retail CDs in the past. How do you want to tackle this issue? Everyone who installs TombATI gets the "pirated" tracks with that. When we break off the TombATI dependency, people will install T1M on their Steam versions… then what? "T1M bug: the game doesn't play any music!" "Wait, please run this tool.exe hosted on my private server that will rip the tracks from your CD. Don't have a CD drive? No worries, here's a link to Amazon for cheap offers." I think you will agree that this is less than ideal.

And BTW let's not kid ourselves, reverse engineering is in itself piracy as evidenced by the recent re3 court cases. What's the most funny is that after suing re3 developers for pirating their intellectual property, Take Two released their GTA: Definitive Edition… which contained copyrighted music assets. They had to take the game down themselves to remove those. No court cases (yet). Talk about hypocrisy.…

**Comment by rr-** [comment]
Linking #188 here and closing the issue for it because it is very much related, and I think needs more polishing.

**Comment by rr-** [comment]
#216 is about the same thing as well.

**Comment by Richard-L** [comment]
Thanks for responding!

> TombATI dependence

Indeed I wasn't aware of everything currently required from TombATI. This was mostly trying to focus on assets like level data, music, sfx,  images or videos.

I like the three parts you listed – they make good sense. Best of success with your current efforts to lose TombATI dependency. I really wish I could help out with this more directly myself.

> Tables

Have you obtained the games through Steam/GOG? These will be the files you get. I am focusing more on the original retail data because anything else is just a re-bundle in one way or another.

> Lazy people

I agree completely. I was honestly just alluding to what I assumed the stereotypical dev attitude would be. Good to see yours is different.

What I think works well UX wise are drag'n'drop windows as an importer, where people can just drop any files they have without needing to know what they're for. Only after it could parse the essential files to run TR1M, it outputs the game's executable. This avoids people attempting to run the game and running into errors. I can happily come up with an easy to understand UI mapping out asset import progress, requirements, and indications what further optional assets can be added to unlock a more complete TR1 experience.

> Steam version / Piracy

If a source people have, whether that be Steam or not, doesn't come with all assets (eg music), then that's what it is. What's available is outside the scope of the project, and it should be, so you don't have to get your hands dirty. You play TR1M without the music tracks in this case, as you would have on the PC back in the day. The importer would signal it well that there is more available to the game, if respective sources are provided. If you wanted to further counter "Music is missing" user complaints, you could show a summary of available assets inside the main menu, or even trigger OSD messages telling of a missing track in the very moment. Feel free to disagree, but I would say it's more important to not per se include any copyrighted assets with the project. And indeed the importer would need to rip audio tracks once. I know all this is less than ideal, but I think the least "piratey" way possible. My main concern with all this is that I would much hope for the project to gain significant following and popularity, and a big enough audience where these concerns will actually start to matter. And it seems necessary to address them at the beginning.

I'm not familiar with the RE3 cases. You will know these aspects better than me. But to my layman's understanding the dumping of files from one's own legally bought copies should seem better than to just include pirated ones.

**Reply by rr-** [reply]
I agree with the sentiment to keep this project clean and not mix it with assets. But we cannot dismiss people from GOG and Steam; I am fairly confident that a decisive majority of the players and streamers belong to this group.

I suggest we make the asset importer into a completely separate project, Tomb1Main-Installer, that offers to import the game from GOG or Steam (and your CDs), and we host the music files somewhere. Then, the shady script would download this music from our secret mirrors if needed? And if the mirrors go down, too bad.

Another option would be to just link to the missing assets like now. It's not like this repository hosts them. And in the end it's not that different because the repositories will be associated either way.

I am used to working in the gray zone. The only reason we are having this discussion is because the actual owners of the TR IP cannot package their own game properly. If DMCAs come our way, I'm fine with taking it down (the installer) and letting people torrent it. Plus like I said: if Square wanted to do so, they could nuke this repository any second they wanted (even without the links), and any other of our reversing projects for that matter.

**Reply by Richard-L** [reply]
I think a separate installer project is a great idea :+1: 

Why do you say GOG/Steam people would be dismissed? Couldn't they just point the installer to the files they got from either service? If you mean music wise, I can see where you're coming from, but I don't see a way around it. Clever users will know it takes less than minutes to download images of all the console releases. The more genuine approach would be to buy them on ebay or similar.

You're not a fan of ripping audio tracks from physical or virtual drives, are you? Why not if I may ask? Alternatively people could be advised to dump their legal copies, and perhaps these could be written to memory upon launch without the need to rip anything? Just thinking out loud.

Perhaps for reference, the structure of TR1 audio tracks is identical on all PS1/Saturn releases. The tracks are recognised by most online databases. I use Gracenote, where per release the track titles usually come in 2-3 variants. I'll include the more common ones below:

| Track # | Title | Length | Present on PC release |
| --- | --- | --- | --- |
| 01 | _data_ | ... | ... |
| 02 | Main Theme I | 3:17 | x |
| 03 | Calm I / Mysterious | 1:04 | |
| 04 | Main Theme II | 2:04 | |
| 05 | Cave Echoes / Inside the Tomb | 2:00 | x |
| 06 | Hurried I / The Earth Begins to Shake | 1:17 | |
| 07 | Wonder / The Tall Room | 0:56 | |
| 08 | Chase I / Dangerous Foe | 1:08 | |
| 09 | Found Gem / Intriguing | 0:40 | |
| 10 | Riddle I / The Coliseum | 1:19 | |
| 11 | Effect I / The Door Opens | 0:08 | |
| 12 | Effect II / Another Door Opens | 0:15 | |
| 13 | Effect II | 0:07 | |
| 14 | Effect IV | 0:12 | |
| 15 | Calm II / A Discovery | 0:11 | |
| 16 | Chase II / Dangerous Foe (shortened version) | 0:39 | |
| 17 | Calm III / The Sacred Place | 0:43 | |
| 18 | Calm IV / The Evil Place | 0:25 | |
| 19 | Main Theme III | 1:29 | |
| 20 | Hurried II / Someone is After You | 0:54 | |
| 21 | Riddle II / The Golden Hand of Midas | 0:41 | |

All following tracks, except the 24th of Natla installing the scion, are localisation dependent. There are four: **English, German, French, Japanese**. The only difference between the PS1 and Saturn versions is how Lara addresses the controller buttons in her gym lines. The Russian fan community have later added an unofficial Russian dub that is also circulating.

The following 4 tracks are played during the rendered cutscenes. Curiously they are not in order of game chronology:

| Track # | Title | Length | Present on PC release |
| --- | --- | --- | --- |
| 22 | Dialogue (main): Back again... / The Hatching | 1:04 | x (not on UB) |
| 23 | Dialogue (main): Well you... / Lara vs. Larson | 0:58 | x (not on UB) |
| 24 | Dialogue (main): (Dramatic Sounds) / Sinister | 0:15 | x (not on UB) |
| 25 | Dialogue (main): Here lies... / Lara reads Tihocan eulogy | 0:37 | x (not on UB) |

The gym tracks follow. They are all in correct concession. The PC versions have these attached as SFX to `GYM.PHD`:

| Track # | Title | Length |
| --- | --- | --- |
| 26 | Dialogue: Welcome to... | 0:07 |
| 27 | Dialogue: Use the... | 0:07 |
| 28 | Dialogue: Okay let's... | 0:07 |
| 29 | Dialogue: Now press... | 0:07 |
| 30 | Dialogue: Ar, the... | 0:08 |
| 31 | Dialogue: Run up... | 0:07 |
| 32 | Dialogue: This used... | 0:10 |
| 33 | Dialogue: I don't... | 0:10 |
| 34 | Dialogue: With the... | 0:08 |
| 35 | Dialogue: If you... | 0:07 |
| 36 | Dialogue: If a... | 0:17 |
| 37 | Dialogue: Press forward... | 0:07 |
| 38 | Dialogue: If I... | 0:07 |
| 39 | Dialogue: Walk to... | 0:16 |
| 40 | Dialogue: Right, this... | 0:11 |
| 41 | Dialogue: Nice! | 0:07 |
| 42 | Dialogue: Try to... | 0:07 |
| 43 | Dialogue: I can't... | 0:09 |
| 44 | Dialogue: Great, if... | 0:08 |
| 45 | Dialogue: Tap backwards.... | 0:10 |
| 46 | Dialogue: Then let... | 0:07 |
| 47 | Dialogue: Let's go... | 0:07 |
| 48 | Dialogue: The jump... | 0:07 |
| 49 | Dialogue: Woah, air... | 0:16 |
| 50 | Dialogue: Right, now... | 0:07 |

Lastly, there are the short notions spoken by the characters during some of the levels, again in random order. On the PC these are again part of the respective `.PHD`:

| Track # | Title | Length |
| --- | --- | --- |
| 51 | Dialogue (male): Say cheese! | 0:07 |
| 52 | Dialogue (male): Ain't nuthin... | 0:07 |
| 53 | Dialogue (male): I still... | 0:09 |
| 54 | Dialogue (female): You can't... | 0:07 |
| 55 | Dialogue (male): A little... | 0:10 |
| 56 | Dialogue (male): You firin'... | 0:07 |

The rest of the CDs is filled by a silent 57th track.

So the main desire would be what on console releases are tracks 03-04 and 06-21, meaning 18 tracks total. Let's not get into the different audio qualities of the dialogue tracks here, which of course are higher when they're console audio tracks which were produced first, before they were turned into PC SFX assets.

On the PC's side, depending on whether you insert a TR1 base game CD, UB standalone disc, or a "Gold" bundle, you'll be able to add these three tracks:

- Wind Ambience (not included on UB)
- Water Ambience (not included on UB)
- Heartbeat Ambience

They are exchanged for the Caves ambient in some levels.

All this is going into quite some detail already regarding what intelligence would be required of an importer to tell all this apart across several releases to be thrown at it. It's just to illustrate and capture this information a little. The same could be done for video, image and level assets.

**Reply by rr-** [reply]
Thanks for the compilation.

>You're not a fan of ripping audio tracks from physical or virtual drives, are you?

My personal preference here is not the issue. The issue is that you're telling 99% of the people that want to play TR1 these days to go buy old disks from the aftermarket, when they have already paid their money for their version on Steam/GOG. There is limited number of such CDs in circulation. [CDs have limited lifespan](https://www.clir.org/pubs/reports/pub121/sec4/). Desktop computers do not ship with CD ROM drives anymore, yet alone laptops.  And I'll repeat this: Steam version is only crippled because of Square's negligence. Including music one way or another, piracy or not, is fixing their mistakes really. I'll think about this more but currently I'm leaning towards keeping a link to a third party service where people can download English audio, and will deal with potential DMCAs as they come (I've dealt with such in the past). And I'd rather people not use some shady torrents because of two things:
1) They'll be pirating their games for real, with all the assets and whatnot.
2) We'll have to deal with tech support requests when those torrents do not work for them.

**Reply by Richard-L** [reply]
All makes good sense. Let's sit on it.

Just this: I'd totally make it a rule of support that any piracy talk is prohibited, and have 2-3 mods who take care of it. But yeah, ideally this problem of "console asset exclusivity" didn't even exist in the first place.

**Comment by Richard-L** [comment]
#356 and #358 have triggered me to post some details on the image assets present on the three main TR1 platforms.

Sadly GitHub does really terrible cell spacing in the below tables, and I don't know how to fix it. But you get the idea.

**PS1** — 384x256px
| # | Filename | MD5 | Date Modified | File Size | PNG conversion | EU | US | JP |
| --- | --- | --- | --- | --- | --- | --- | --- | --- |
| 01 | AMERTIT.RAW | 7A4B00E4A73CC248DA760B0279A9C9D7 | 1996-10-09 05:18:00 | 69,633 | ![AMERTIT](https://user-images.githubusercontent.com/4934209/150838162-9dd44009-5a54-405e-95de-6c86730d2f7f.png) | x | x | |
| 02 | ATLANLOA.RAW | 688ECBBA607B8FAD52C061A5A0DE0E51 | 1996-10-02 11:03:50 | 45,218 | ![ATLANLOA](https://user-images.githubusercontent.com/4934209/150837797-aaef3d64-4e9c-4097-b83d-1b0c0d52689d.png) | x | x | x |
| 03 | AZTECLOA.RAW | 0BBCBB88378A8450DAC69AE32CB38168 | 1996-10-02 11:03:58 | 70,938 | ![AZTECLOA](https://user-images.githubusercontent.com/4934209/150838329-945da653-28c0-4f34-a0c6-254316bc2c0b.png) | x | x | x |
| 04 | CRED.RAW | 20785D1CBF0F5441DB0DCD9C3306D77B | 1996-09-26 05:44:22 | 20,033 | ![CRED](https://user-images.githubusercontent.com/4934209/150838451-daedc272-704b-40a7-93ff-a3a15737fa4d.png) | x | x | x |
| 05 | CRED1.RAW | CF07FB04D6E45879651EDA5D062F3C2B | 1996-11-20 07:54:24 | 55,430 | ![CRED1](https://user-images.githubusercontent.com/4934209/150887872-bdc2965a-a5b2-42ce-9226-74d26f4d14a8.png) | | | x |
| 06 | CRED2.RAW | BA9BFE0A5E1C408CC6B852E108C2B091 | 1996-11-20 07:55:16 | 63,528 | ![CRED2](https://user-images.githubusercontent.com/4934209/150887996-70185d32-263c-4331-92dc-3b5813d506c2.png) | | | x |
| 07 | CRED3.RAW | 64C019A2F693D8DDC7EF6F997D8C5187 | 1996-11-20 07:55:26 | 64,178 | ![CRED3](https://user-images.githubusercontent.com/4934209/150888098-ff9d738c-a934-42f2-8114-e2a0cb89db24.png) | | | x |
| 08 | CRED4.RAW | B16CDF0A68EFFB89F5670D7981F18A43 | 1996-11-25 08:23:40 | 25,802 | ![CRED4](https://user-images.githubusercontent.com/4934209/150888174-279aa663-72b9-4d8f-bfdd-3d226162d70e.png) | | | x |
| 09 | CRED5.RAW | 36725AE185C751AEE86598D6E0FB62E5 | 1996-11-25 08:24:00 | 12,555 | ![CRED5](https://user-images.githubusercontent.com/4934209/150888344-84a28c12-80ec-4490-a4bb-6bbbfe190e28.png) | | | x |
| 10 | EGYPTLOA.RAW | 1ED837C22E094865B4E86821C5B779B5 | 1996-10-02 11:04:08 | 78,214 | ![EGYPTLOA](https://user-images.githubusercontent.com/4934209/150838662-f847fe21-1fb2-4fbf-abf5-22161afbd020.png) | x | x | x |
| 11 | END.RAW | 81EDC8F66AC196FF16B930766A06EBC5 | 1996-09-26 05:44:50 | 23,600 | ![END](https://user-images.githubusercontent.com/4934209/150838681-dd25f43e-38f5-4fc5-b306-f3beef478022.png) | x | x | x |
| 12 | GREEKLOA.RAW | 763256EFAC2DAC6AC922C39E2B1890F6 | 1996-10-02 11:04:16 | 96,657 | ![GREEKLOA](https://user-images.githubusercontent.com/4934209/150838767-4d062076-55a5-4064-b391-319f43634e9d.png) | x | x | x |
| 13 | GYMLOAD.RAW | 15313A6C0AD54B7B423089EF91E0738B | 1996-10-02 11:04:28 | 79,409 | ![GYMLOAD](https://user-images.githubusercontent.com/4934209/150838806-c3b6587c-a384-4226-bcc9-bac89834655c.png) | x | x | x |
| 14 | JAPTIT.RAW | 7A68A46F2CE1836BC4CDCF4B4FD50189 | 1996-12-06 07:02:30 | 23,373 | ![JAPTIT](https://user-images.githubusercontent.com/4934209/150888643-d299405e-3953-4b92-8089-3d18f7f7b6a7.png) | | | x |
| 15 | PSXTIT.RAW | 888CB94300C41DBCBF254C9A03D460C6 | 1996-09-26 05:45:30 | 70,996 | ![PSXTIT](https://user-images.githubusercontent.com/4934209/150838822-2d6aa7be-9aeb-4c77-8238-a936c7c4a5d5.png) | x | | |

Notes:
- 05, 07 and 08, and then also 12 and 13 are unique to PS1
- 09 I've only included for completeness' sake. Only contains jap staff names

——————————————————

**Saturn** — 320x256px or 352x224px
The US release has these in NTSC's lower 320x224px resolution. Curiously the Japanese release includes _both_ the PAL and NTSC images. The Japanese release adds nothing of interest other than the above credit screens in lower resolution than the PS1 has. Thus I'm only going to include EU for an overview.

| # | Filename | MD5 | Date Modified | File Size | PNG conversion |
| --- | --- | --- | --- | --- | --- |
| 01 | ATLANTIA.BIN | FB6A99016CF1C6A46BB90E209C20C1DD | 1996-09-17 06:50:29 | 163,840 | ![ATLANTIA](https://user-images.githubusercontent.com/4934209/150840751-ae9bc936-7184-42ec-b41a-1f8b71cdc42f.png) |
| 02 | AZTEC.BIN | 6EEE92FFABC5571C9817EE5DE924A2CC | 1996-09-17 06:50:29 | 163,840 | ![AZTEC](https://user-images.githubusercontent.com/4934209/150840849-574b36f2-2e59-4f23-968c-8d8abffb3c27.png) |
| 03 | CREDITS.BIN | 239D9CFDACF7DA7A5D3BCF31F7D71842 | 1996-09-17 06:50:29 | 180,224 | ![CREDITS](https://user-images.githubusercontent.com/4934209/150840940-26b4a2da-550c-44ec-b0c0-32f58e1c50e2.png) |
| 04 | EGYPT.BIN | 09198EC9316FA097C4F96210DDCADA31 | 1996-09-17 06:50:29 | 163,840 | ![EGYPT](https://user-images.githubusercontent.com/4934209/150841015-5b930a9a-4b73-42c3-8b52-562f2e62199b.png) |
| 05 | ENDPAGE.BIN | 6F4CBE1F23916DF56CE6835066B9A9FB | 1996-09-17 06:50:29 | 157,696 | ![ENDPAGE](https://user-images.githubusercontent.com/4934209/150841100-74566386-ef13-4f0a-85e8-78a0829e4575.png) |
| 06 | ENGTIT1.BIN | 8B88BA0DAC88B8140F308DB1CEE3E664 | 1996-09-17 06:50:29 | 157,696 | ![ENGTIT1](https://user-images.githubusercontent.com/4934209/150841184-70d3457a-47b1-4f9d-868e-b736afa950bb.png) |
| 07 | GREEK.BIN | B7F87CF9E0D0839A70CB396003250C44 | 1996-09-17 06:50:29 | 163,840 | ![GREEK](https://user-images.githubusercontent.com/4934209/150841247-34738976-2ccc-415f-9d38-d9082c521dc9.png) |
| 08 | GYM.BIN | D2E682A7E4EF4F8F84FD9F69468D4120 | 1996-09-17 06:50:29 | 163,840 | ![GYM](https://user-images.githubusercontent.com/4934209/150841311-bf15efbb-f479-48e6-9470-31bddae81468.png) |
| 09 | LOADING.BIN | 314222AF290DDC6026283BF26E865F20 | 1996-09-17 06:50:29 | 157,696 | ![LOADING](https://user-images.githubusercontent.com/4934209/150841387-b514d2e1-b2e8-4475-8fca-cab104def393.png) |
| 10 | SATOPT.BIN | 40D43CFE7B8E4FEF187CBFED2068CB93 | 1996-09-17 06:50:29 | 180,224 | ![SATOPT](https://user-images.githubusercontent.com/4934209/150841594-d03cbe07-db85-4de7-ac11-4759bc7d7773.png) |
| 11 | SEGALOGO.BIN | 9E08AABA55C1741DFF2ABD8BBEFC6F97 | 1996-09-17 06:50:29 | 143,360 | ![SEGALOGO](https://user-images.githubusercontent.com/4934209/150841721-f40f8095-09f2-4687-b112-a7191044c066.png) |
| 12 | TITLE1.BIN | 682C0C77672885FD01679FB6A7E7A039 | 1996-09-17 06:50:29 | 180,224 | ![TITLE1](https://user-images.githubusercontent.com/4934209/150841827-b794438d-e914-431d-833d-a4d47a15b01f.png) |
| 13 | TITLE2.BIN | 314222AF290DDC6026283BF26E865F20 | 1996-09-17 06:50:29 | 157,696 | ![TITLE2](https://user-images.githubusercontent.com/4934209/150841906-e52c5b13-d0d8-456d-bfbc-036ce64233ff.png) |

Notes:
- 05 afaik was never used on the retail release. I believe this showed at the end of one of the Saturn TR1 demos, and for some reason found its way into retail. It's also on the Japanese version, in kana.
- 07 and 08 are unique to Saturn.
- 09 and 13 are identical. Not sure why they kept a duplicate.

——————————————————

**PC** — 640x480px
| # | Filename | MD5 | Date Modified | File Size | PNG conversion | TR1 | UB |
| --- | --- | --- | --- | --- | --- | --- | --- |
| 1 | cred0.pcx | 9BECC9EFBDDD9B24D4687643135EEA49 | 1998-02-03 17:10:38 | 231,538 | ![cred0](https://user-images.githubusercontent.com/4934209/150894358-60dc9e4e-72d3-441c-b4df-e0d3293ea439.png) | | x |
| 2 | CRED1.PCX | 6272C2A557CD7597EFC6594E86AE7F2F | 1996-10-18 12:40:32 | 106,587 | ![CRED1](https://user-images.githubusercontent.com/4934209/150877822-7f4c496f-b5dc-4d23-939d-4ed8f86e79f8.png) | x | x |
| 3 | CRED2.PCX | 4497435DD41ABD9A247BE6AFBF264769 | 1996-10-18 12:40:52 | 196,985 | ![CRED2](https://user-images.githubusercontent.com/4934209/150877883-87db07b9-aa5f-4d74-b179-394bc3905c23.png) | x | x |
| 4 | CRED3.PCX | 9DF86E1D2523E3053D07B51CEBF33143 | 1996-10-18 12:42:04 | 174,908 | ![CRED3](https://user-images.githubusercontent.com/4934209/150881615-1887f4bf-1da2-431b-ab90-5ae1ff490c48.png) | x | x |
| 5 | EIDOSPC.PCX | B2815D0CC7E8DA83E0A2033A88943D82 | 1996-10-24 18:04:20 | 22,700 | ![EIDOSPC](https://user-images.githubusercontent.com/4934209/150881681-5b74d16e-b852-42eb-b0d6-a7914127c5f4.png) | x | x |
| 6 | END.PCX | 8B678E0162D158C28B7911EF091C6C41 | 1996-10-18 12:42:36 | 102,512 | ![END](https://user-images.githubusercontent.com/4934209/150881788-f49bc174-bc48-42ef-a91d-eda06422567c.png) | x | x |
| 7 | INSTALL.PCX | 2F4F3EAFF7C99D9F16A530AFB4ECC3B1 | 1996-10-18 14:32:24 | 168,100 | ![INSTALL](https://user-images.githubusercontent.com/4934209/150881894-15fae7dc-b293-464f-a05f-1afa93a781d9.png) | x | x |
| 8 | TITLEH.PCX | 44C7DA52B6DD400F614C7845E6F05DA3 | 1996-10-08 14:54:42 | 115,980 | ![TITLEH](https://user-images.githubusercontent.com/4934209/150881945-7a08c5ec-60da-4b85-87a6-fbc355898967.png) | x | |
| 9 | TITLEH.PCX | 3DFBCE6C717400AC8AE9241D3FD25D9C | 1998-01-30 12:13:02 | 73,715 | ![TITLEH](https://user-images.githubusercontent.com/4934209/150894833-3f78b59e-a06b-44c7-b44a-e6d509cb4056.png) | | x |

Notes
- These are mostly all used by TR1M at this point, or have been replaced with optional fan-upscales by default.
- 1 is the added little credit screen the creators of UB made for themselves.
- 9 is UB's title screen which unfortunately kept the same filename as TR1's.
- 1, 4, 7 and 9 are unique to PC.
- All the images to be found on the consoles too are usually to be preferred from PC, simply because they came in higher res.

__**NB**__: Some of the [demos/betas/prototypes](https://www.tombraiderforums.com/showthread.php?t=207170) for both platforms contain slightly different image assets, which consequently are a topic of fan interest. It would be an even longer job to make comparisons of all of them, and one I doubt would bring much novelty. EDIT: Thankfully though, the ever amazing SuikazeRaider has done so, and made his results available [here](https://www.tombraiderforums.com/showthread.php?t=113082).

__**NB2**__: It is worth noting that on this site by Core Design, many of these image assets have been released in public, sometimes in interesting variants or higher quality versions: https://core-design.com/goodies_tr1.html

**Reply by Richard-L** [reply]
@rr- there are a few more image assets from realtech VR's android conversion in 2013. They contain "cleaned" versions of some of the PC images above, and some more localisation backdrops for UB. I don't feel comfortable to just upload them here. Is there a way I can show them to you privately?

**Reply by rr-** [reply]
Sure :+1: you can mail me them at dash@wind.garden .

**Comment by walkawayy** [comment]
@rr- posting this here since I think it fits best. Someone in the Discord was asking if we could include an optional addition to the installer to download the PS1 FMVs since they are higher quality than the PC. I don't remember the conclusion of the asset distribution issue, but I know the installer can download the music files separately to keep the installer size down. Is this possible for the PS1 FMVs? LMK your decision and I can open an issue for it if you think we could support it.

**Reply by rr-** [reply]
I think it's a good idea :green_circle:
